### ISSUE TITLE
Support for Moonlander 2 Scrypt Miner

### ISSUE THREAD

**Issue by bertstoner** [nwoolls/MultiMiner#334]
Subject says it. I can't seem to get the latest version of MultiMiner to recognize my moonlander 2 miners. I'd say it's driver related.  Any and all help is appreciated.

**Comment by nwoolls** [comment]
@bertstoner try putting a version of BFGMiner that supports that ASIC in `<MultiMiner>\Miners\BFGMiner` and then scanning for hardware.

**Comment by nwoolls** [comment]
@bertstoner I've confirmed that placing JStefanop's fork of BFGminer into the `<MultiMiner>\Miners\BFGMiner` folder does work. Let me know if you have any questions or issues. Once the driver is merged into the official BFGMiner release the support in MultiMiner will be OOTB.
